### PR TITLE
Add acceleration to offboard plugin

### PR DESCRIPTION
**Pull request by JonasVautherin**
Depends on https://github.com/mavlink/MAVSDK-Proto/pull/216

### CODE REVIEW

File: src/integration_tests/offboard_acceleration.cpp [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+    // Stop to stabilize
+    {
+        offboard->set_acceleration_ned(still);
+        std::this_thread::sleep_for(std::chrono::seconds(3));
+    }
+
+    // Go west for a few seconds
+    {
+        Offboard::AccelerationNed go_west{};
```
Comment (julianoes) [inline src/integration_tests/offboard_acceleration.cpp:105]: Go West, live is peaceful there, go West ... :notes:

File: src/plugins/offboard/offboard_impl.cpp [hunk 2]
```diff
@@ -437,6 +465,46 @@ Offboard::Result OffboardImpl::send_position_velocity_ned()
...
+        0.0f, // x,
+        0.0f, // y,
+        0.0f, // z,
+        0.0f, // vfx
+        0.0f, // vfy
+        0.0f, // vfz
+        acceleration_ned.north_m_s2,
+        acceleration_ned.east_m_s2,
+        acceleration_ned.down_m_s2,
+        0.0f, // yaw
```
Comment (julianoes) [inline src/plugins/offboard/offboard_impl.cpp:502]: What about `yaw`? You are not ignoring it but setting it to 0.0?

File: src/plugins/offboard/offboard_impl.cpp [hunk 3]
```diff
@@ -437,6 +465,46 @@ Offboard::Result OffboardImpl::send_position_velocity_ned()
...
+    mavlink_message_t message;
+    mavlink_msg_set_position_target_local_ned_pack(
+        _parent->get_own_system_id(),
+        _parent->get_own_component_id(),
+        &message,
+        static_cast<uint32_t>(_parent->get_time().elapsed_s() * 1e3),
+        _parent->get_system_id(),
+        _parent->get_autopilot_id(),
+        MAV_FRAME_LOCAL_NED,
+        IGNORE_X | IGNORE_Y | IGNORE_Z | IGNORE_VX | IGNORE_VY | IGNORE_VZ | IGNORE_YAW_RATE,
```
Comment (TSC21) [inline src/plugins/offboard/offboard_impl.cpp:492]: should we add `IGNORE_YAW` as well?

Comment (JonasVautherin) [reply src/plugins/offboard/offboard_impl.cpp:492]: aha maybe that's needed, let me do more tests

**Comment by JonasVautherin** [comment]
For the record, here is what I get flying the integration test. Not completely sure about the change in altitude...

2021-03-12-112008_1831x1202_scrot

**Comment by TSC21** [comment]
> For the record, here is what I get flying the integration test. Not completely sure about the change in altitude...
> 
> 2021-03-12-112008_1831x1202_scrot

@Jaeyoung-Lim FYI

**Comment by JonasVautherin** [comment]
According to Matthias, the drift on z is expected :+1:.

**Review by TSC21 — APPROVED** [review]
LGTM

**Review by julianoes — APPROVED** [review]
Looks fine now.

**Pull request merged**